src/app.rs

```diff
@@ -53,6 +53,13 @@ pub fn build() -> App<'static, 'static> {
                 .number_of_values(1)
                 .help("Whether to use fancy or unicode icons"),
         )
+        .arg(
+            Arg::with_name("indent")
+                .short("i")
+                .long("indent")
```

Comment:
There is a lot of possible indentations. Specifying `prefix-indent` would avoid conflicts with some possible future features. It's longer but I guess that this flag will be only used inside the alias so it shouldn't be too inconvenient.

@meain what do you think of it?

Reply:
It is a little long. I'd prefer to find a shorter flag, but I can't think of any at the moment.

Reply:
We should probably reserve the short `i` flag, maybe for something like ignore which will be a more important flag.

I think we should go with `prefix-intent`. Yes, it is a bit long, but I feel like this will not be a common flag.